netbox/extras/forms/model_forms.py

```diff
@@ -301,6 +304,66 @@ def __init__(self, *args, initial=None, **kwargs):
         super().__init__(*args, initial=initial, **kwargs)
 
 
+class TableConfigForm(forms.ModelForm):
+    slug = SlugField()
+    object_type = ContentTypeChoiceField(
+        label=_('Object type'),
+        queryset=ObjectType.objects.all()
+    )
+    ordering = SimpleArrayField(
+        base_field=forms.CharField(),
+        required=False,
+        label=_('Ordering'),
+        help_text=_(
+            "Enter a comma-separated list of column names. Prepend a name with a hyphen to reverse the order."
```

Comment:
My observation here is that the name of the column in case-insensitive. Is this worth mentioning here or no?

Reply:
The list should be case-sensitive. Each item needs to be an exact match to a column name (the name of the column itself, not its rendered verbose name/title).

Reply:
Ah, I bet that's what was causing my issue with the first item not being picked up/used.